### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by victory-nwawunze**
I just created the landing page, and It is 100% responsive.

### CODE REVIEW

File: public/assets/tumbs up 1.svg
Comment (onyegood) [inline public/assets/tumbs up 1.svg:1]: Assets file must not have space in its name, remove spaces and name it properly

File: public/assets/facebook icon.svg
Comment (onyegood) [inline public/assets/facebook icon.svg:1]: This asset should be called facebook, remove the space and icon.

File: common/mobileApp/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <div>
+              <h2 className='text-5xl font-semibold text-white'>Learn on the go!</h2>
+              <p className='pt-6 text-white'>
+                download the mobile app to
+                <br /> start learning.
+              </p>
+              <div className='mt-6 flex items-center'>
+                <div className='mr-6'>
+                  <Image src='/assets/google.svg' width={150} height={150} alt='Banner Image' />
+                  {/* <Image src='/assets/google.svg' width={20} height={20} /> */}
```
Comment (onyegood) [inline common/mobileApp/index.tsx:19]: What is this comment doing here?

File: common/mobileApp/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                </div>
+                <div>
+                  <Image src='/assets/apple.svg' width={150} height={150} alt='Banner Image' />
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div className=' mt-8 rounded-lg'>
+            {/* Img */}
```
Comment (onyegood) [inline common/mobileApp/index.tsx:29]: What is this comment doing here?

File: common/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+              open and ready to hear froms.{' '}
+            </p>
+            <Button className='get-btn mt-6 font-bold shadow-xl'>Get In Touch</Button>
+          </div>
+          <div className=' mb-[30%]  p-4 '>
+            <h2 className=' font-normal text-black'>Email:</h2>
+            <p className='mb-6 mt-6 text-4xl font-semibold text-black '>efikokids@gmail.com</p>
+            <h2 className='mb-4 font-semibold text-black'>Phone:</h2>
+            <p className='text-3xl font-semibold text-black  '>+234-7031-0020-66</p>
+            <div className='mt-[15%] flex items-center'>
```
Comment (onyegood) [inline common/Footer/index.tsx:23]: Create a component called `SocialMedia` and pull all social media icons there.

File: common/Footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+              <div className='mr-14'>
+                <Image src='/assets/instagram.svg' width={60} height={40} alt='instagram' />
```
Comment (onyegood) [inline common/Footer/index.tsx:25]: Make this a reusable component

File: common/Footer/Footer.scss
Comment (onyegood) [inline common/Footer/Footer.scss:1]: We must not hardcode color anywhere in the app. Also please remove `!important` anywhere you have it in your CSS file.

File: common/Banner/banner.scss [hunk 5]
```diff
@@ -0,0 +1,27 @@
+.banner {
+  .banner-text {
+    color: #c505cc;
+
+    font-size: 3rem;
+  }
+  /* .pl-30 {
```
Comment (onyegood) [inline common/Banner/banner.scss:7]: Remove this comment

File: common/Desc/Desc.scss [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.desc {
+  .my-12 {
+    margin-top: 13% !important;
+  }
+  span {
+    padding-top: 20px;
+  }
+  /* spacing {
```
Comment (onyegood) [inline common/Desc/Desc.scss:8]: Remove commented block

File: common/Desc/index.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+            Because foundation is the key of every building, <br /> we channel our energy mostly on pre-school and
+            primary.
+          </p>
+        </div>
+      </section>
+      <section className='desc '>
+        <div className='bg-white text-center'>
+          <h1 className='my-6  mt-[13%]  text-4xl text-black'>
+            Why<span className='font-extrabold text-pink-600'> efiko kids?</span>
+            {/* We are majorly focused on <br /> <span className=' font-bold'>two different grades</span> */}
```
Comment (onyegood) [inline common/Desc/index.tsx:22]: Remove the commented block.

File: app/globals.css
Comment (ayotidapo) [inline app/globals.css:1]: leave the background color as it is in the global.scss file, you can give the homepage its own container background color

**Review by ayotidapo — CHANGES_REQUESTED** [review]
a

File: app/globals.css [hunk 8]
```diff
@@ -34,7 +34,7 @@
 body {
   height: 100%;
   color: var(--text-color);
-  background: #f3f4f5;
+  background: white;
```
Comment (ayotidapo) [inline app/globals.css:37]: leave the background color as it is in the global.scss file, you can give the homepage its own container background color

**Review by ayotidapo — CHANGES_REQUESTED** [review]

File: common/Desc/Desc.scss (hunk 6 above)
Comment (ayotidapo) [reply common/Desc/Desc.scss:8]: and !important too

File: common/Footer/Footer.scss [hunk 9]
```diff
@@ -0,0 +1,6 @@
+.get-btn {
+  background-color: #c505cc !important;
```
Comment (ayotidapo) [inline common/Footer/Footer.scss:2]: lets agree on @onyegood suggestion of using var for colors and avoiding !important

File: common/homeNavbar/Navber.scss [hunk 10]
```diff
@@ -0,0 +1,6 @@
+.sign-in-btn {
+  background-color: white !important;
```
Comment (ayotidapo) [inline common/homeNavbar/Navber.scss:2]: !important... Mbah!

File: common/homeNavbar/index.tsx [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+              Buy learning tab
+            </a>
+            <a href='#' className='px-5 text-sm text-black'>
+              Buy subscription
+            </a>
+            <a href='#' className='px-5 text-sm text-black'>
+              Buy Efiko Swag
+            </a>
+          </div>
+          <Button className='sign-in-btn font-bol w-1.5 rounded  shadow-lg'>Sign in</Button>
```
Comment (ayotidapo) [inline common/homeNavbar/index.tsx:34]: is font-bol tailwind? I hope its not typo

File: common/plans/Plans.scss [hunk 12]
```diff
@@ -0,0 +1,20 @@
+.btn-style {
+  background-color: white !important;
```
Comment (ayotidapo) [inline common/plans/Plans.scss:2]: lets avoid or totally not use !important

File: common/plans/index.tsx [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+const SubPlans = () => {
+  return (
+    <div className='bg-[#F4F3F6]  '>
+      <h1 className='pb-24 pt-36 text-center text-4xl text-black'>
+        Subscription <span className='font-bold text-[#067CFF]'>plan</span>
+      </h1>
+      <div className='p-6 md:p-6 lg:p-6'>
+        <div className='grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3'>
+          <div>
+            <div className='rounded-lg bg-[#EF03F8] p-4 shadow-md'>
```
Comment (ayotidapo) [inline common/plans/index.tsx:13]: Any tailwind custom properties used more than 2 times means its important and might likely to be used later lets start adding it to our tailwind extends configuration or better still add it to css variable, either of the two ways is fine

**Review by ayotidapo — CHANGES_REQUESTED** [review]

File: public/assets/linked in.svg
[the remark this answers is not in the document]
Comment (victory-nwawunze) [reply public/assets/linked in.svg:1]: I have removed the space in all images.

File: public/assets/facebook icon.svg
Comment (victory-nwawunze) [reply public/assets/facebook icon.svg:1]: I have changed the asset name to Facebook.

File: common/Footer/index.tsx (hunk 4 above)
Comment (victory-nwawunze) [reply common/Footer/index.tsx:25]: It is now reusable, passed in a couple of props and I map over all socialMedia icons.

File: common/homeNavbar/Navber.scss (hunk 10 above)
Comment (victory-nwawunze) [reply common/homeNavbar/Navber.scss:2]: i have Remove the important

File: common/homeNavbar/index.tsx (hunk 11 above)
Comment (victory-nwawunze) [reply common/homeNavbar/index.tsx:34]: It was a typo, i have correct it.

File: common/Footer/Footer.scss
Comment (victory-nwawunze) [reply common/Footer/Footer.scss:1]: Have removed all important in the css

File: common/plans/Plans.scss (hunk 12 above)
Comment (victory-nwawunze) [reply common/plans/Plans.scss:2]: i Have removed !important ineverywhere in the app.

File: package.json
[the remark this answers is not in the document]
Comment (victory-nwawunze) [reply package.json:1]: i Have uninstalled react-icon

File: common/Footer/Footer.scss
Comment (victory-nwawunze) [reply common/Footer/Footer.scss:1]: all styles are now reusable.

Comment (victory-nwawunze) [reply common/Footer/Footer.scss:1]: Updated some SCSS styles to tailwind CSS.

**Pull request closed without merging**